src/tyr/serializers.cc

```diff
@@ -95,12 +95,28 @@ void route_references(json::MapPtr& route_json, const TripRoute& route, const Op
   }
   json::ArrayPtr references = json::array({});
   for (const TripLeg& leg : route.legs()) {
+    references->reserve(references->size() + leg.node_size() - 1);
     for (const std::string& openlr : openlr_edges(leg)) {
       references->emplace_back(openlr);
     }
   }
   route_json->emplace("linear_references", references);
 }
+
+void openlr(const valhalla::Api& api, int route_index, rapidjson::writer_wrapper_t& writer) {
```

Comment:
since i converted the valhalla serializer to rapidjson i needed a separate function to do openlr additions to the response